.github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml

```diff
@@ -0,0 +1,212 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: InitDev (Postgres), Smoke Tests Sandbox, Year-Specific, and Shared Instance
+
+on:
+  pull_request:
+   branches: [main]
+  push:
+   branches: [main]
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+  repository_dispatch:
+    types: [triggered-from-ods-repo]
+
+env:
+  INFORMATIONAL_VERSION: "6.0"
+  BUILD_INCREMENTER: "1118"
+  CONFIGURATION: "Release"
+  VERSION_MAJOR: "6"
+  VERSION_MINOR: "0"
+  CURRENT_BRANCH: ${{ GITHUB.HEAD_REF }}
+  ODS_CURRENT_BRANCH: ${{ github.event.client_payload.ref }}
+  EDFI_SUITE_NUMBER: "Suite3"
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Setup .NET
+      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
+      with:
+        dotnet-version: 6.0.x
+    - name: Support longpaths
+      run: git config --system core.longpaths true
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+            Write-Host "Pattern Name is " $patternName -fore GREEN
```

Comment:
Can you clean up Write host statement every where in Is pull request branch exists in Ed-Fi-ODS-Implementation step

Reply:
This has been removed.